guides/user/kubernetes-v2/run-job-manifest/index.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: single
+title:  "Run Job (Manifest)"
+sidebar:
+  nav: guides
+---
+
+The `Run Job (Manifest)` stage can be used to execute a Kubernetes Job as part of your pipeline. This stage will deploy a `Job` manifest and wait until it completes, allowing you to gate your pipeline's continuation on it's success or failure.
+
+For example use cases, check out [this post](https://blog.spinnaker.io/extending-spinnaker-with-kubernetes-and-containers-5d16ec810d81) on the Spinnaker blog!
+
+## Viewing execution logs
+
+As with any job runner, logs are the primary form of feedback and it's important that viewing them is a first-class experience within Spinnaker. Since users of Spinnaker are all at various stages of Kubernetes adoption we've provided 2 ways to view logs within Spinnaker.
+
+
+### Link to external system
+
+Most Kubernetes deployments will have some utility for forwarding logs from containers and into external systems for log analysis. If you're using one of these platforms, you can configure your Job with the annotation `jobs.spinnaker.io/logs` and a templated URL to your logging system. This value of this annotation will be used to render a link in the UI. 
+
+To make it easier to pinpoint the specific Job, the annotation value can be templated with values from the deployed Job manifest. To use templates, use `{{ templateKey }}` where `templateKey` is the path to the value you wish to use. The deployed manifest will be passed into the template as JSON. This functionality mirrors that of the [Annotation Driven UI](/guides/user/kubernetes-v2/annotations-ui/).
+
+For example, if your Job is deployed with name `myjob-12345`, the annotation
+
+```
+jobs.spinnaker.io/logs: 'https://internal-logging/jobs/{{ metadata.name }}'
+```
+
+will result in the value
+
+```
+https://internal-logging/jobs/myjob-12345
+```
+
+### Within Spinnaker
+
+You can still view the logs of your Job within Spinnaker even if your Kubernetes deployment doesn't forward them to an external system. In the absense of the `jobs.spinnaker.io/logs` annotation, Spinnaker will fetch the deployed manifest and display logs directly in the UI. These logs will only be available for a short period following the Job's completion because Kubernetes only stores events about object for a shot amount of time. If you'd like to view your logs for longer than this timeframe, it's recommended that you use a tool like [Fluentd]() to forward logs to a more persistent platform like Elasticsearch or Datadog.
```

Comment:
```suggestion
You can still view the logs of your Job within Spinnaker even if your Kubernetes deployment doesn't forward them to an external system. In the absense of the `jobs.spinnaker.io/logs` annotation, Spinnaker fetches the deployed manifest and displays logs directly in the UI. These logs are only available for a short period following the Job's completion because Kubernetes only stores events about objects for a short time. If you'd like to view your logs for longer than this timeframe, it's recommended that you use a tool like [Fluentd]() to forward logs to a more persistent platform like Elasticsearch or Datadog.
```